README.md

```diff
@@ -37,12 +37,15 @@ yarn
 # Where errors will be tracked (you may not want to edit this line)
 # SENTRY_URL=
 
-# api base url
-API_BASE_URL=http://...
-
 # OPTIONAL ENV VARIABLES
 # ----------------------
 
+# API base url, fallback to our API if not set
+API_BASE_URL=http://...
```

Comment:
let's comment this line as well, having ... inside it make the parsing crash

Reply:
for the .env i think we need it to move it somewhere else, like a advanced part, or another document that list all env variables. it's not a mandatory thing to set up (i myself never use it) but yet, the README makes it feel it's mandatory!

Reply:
:100: absolutely